plugins/search/webgui/js/searchResult.js

```diff
@@ -33,7 +34,8 @@ function (ObjectStoreModel, BorderContainer, declare, Memory, Observable, topic,
   });
 
   var SearchResult = declare(BorderContainer, {
-    constructor : function () {
+    searchTypes : model.searchservice.getSearchTypes(),
```

Comment:
I get `Uncaught TypeError: Cannot read property 'getSearchTypes' of undefined` for this line.

Reply:
I could not reproduce this error, however, I put the initialization of `searchTypes` into the constructor. Please check if it resolved the error or not.

Reply:
Seems to work for me with this modification.